Content.Client/Fluids/PuddleSystem.cs

```diff
@@ -64,4 +66,34 @@ private void OnPuddleAppearance(EntityUid uid, PuddleComponent component, ref Ap
             args.Sprite.Color *= baseColor;
         }
     }
+
+    #region Spill
+
+    // Maybe someday we'll have clientside prediction for entity spawning, but not today.
+    // Until then, these methods do nothing on the client.
+    public override bool TrySplashSpillAt(EntityUid uid, EntityCoordinates coordinates, Solution solution, out EntityUid puddleUid, bool sound = true, EntityUid? user = null)
+    {
+        puddleUid = EntityUid.Invalid;
+        return false;
+    }
```

Comment:
The xmldoc comments for theses methods should probably mention that `puddleUid` will always be invalid on the client.

Reply:
Good point!